cluster/src/main/java/org/apache/iotdb/cluster/server/member/RaftMember.java

```diff
@@ -2034,6 +2073,10 @@ private long appendEntries(
     }
 
     synchronized (logManager) {
+      if ((logManager.getCommitLogIndex() - logManager.getMaxHaveAppliedCommitIndex())
+          > config.getUnAppliedRaftLogNumForRejectThreshold()) {
+        return Response.RESPONSE_TOO_BUSY;
+      }
```

Comment:
Adding some wait time here may improve robustness.

Reply:
Ok, I add some statements to wait some time before rejection.